android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -484,6 +484,10 @@ public void processProperties(KrollDict d)
 			}
 		}
 
+		if (d.containsKey("linkColor")) {
```

Comment:
There is an iOS-only attribute `ATTRIBUTE_UNDERLINE_COLOR` right now, the parity would be awesome here!

Reply:
https://github.com/tidev/titanium-sdk/blob/58f868b906c10e61fb90e0005938a23dacb5db05/android/modules/ui/src/java/ti/modules/titanium/ui/AttributedStringProxy.java#L264

Should already be there. The label part is for `autoLink` (Android only property) that will convert http links to clickable links